python/feathub/feature_views/tests/test_sliding_feature_view.py

```diff
@@ -404,3 +410,49 @@ def test_build_with_config(self):
 
         self.assertTrue(built_feature.config.get(ENABLE_EMPTY_WINDOW_OUTPUT_CONFIG))
         self.assertFalse(built_feature.config.get(SKIP_SAME_WINDOW_OUTPUT_CONFIG))
+
+
+class SlidingFeatureViewITTest(ABC, FeathubITTestBase):
+    def test_sliding_feature_view_filter_expr(self):
+        df = self.input_data.copy()
+        source = self.create_file_source(df)
+
+        f_total_cost = Feature(
+            name="total_cost",
+            dtype=Int64,
+            transform=SlidingWindowTransform(
+                expr="cost",
+                agg_func="SUM",
+                window_size=timedelta(days=2),
+                step_size=timedelta(days=1),
+            ),
+        )
+
+        expected_result_df = pd.DataFrame(
```

Comment:
Would it be better to put `expected_result_df` after `result_df = ...` than inserting it in the middle of the code that defines `features`?